docs/components/DataList/Web.stories.tsx

```diff
@@ -404,6 +408,309 @@ export const Basic: StoryObj<typeof DataList> = {
   ),
 };
 
+export const ClearAllFilters: StoryFn<typeof DataList> = args => {
+  interface SelectedFilters {
```

Comment:
Yep, that would require either refactoring `Playground.tsx` to work with stories written in CSV3.0, or rewriting stories to to CSV2.0 and making tweaks inside of the stories.
I'm in favour of the first approach, since there's no reason to "go backwards", using old standards/API. But you are right, it is a separate task (or even a set of tasks).